migrations/v5.js

```diff
@@ -0,0 +1,62 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('adapt-contrib-accordion - v4.0.0 > v5.0.0', async () => {
+  let accordions;
+
+  whereFromPlugin('adapt-contrib-accordion - from v4.0.0', { name: 'adapt-contrib-accordion', version: '<=5.0.0' });
+
+  whereContent('adapt-contrib-accordion - where accordion', async content => {
+    accordions = content.filter(({ _component }) => _component === 'accordion');
+    if (accordions) return true;
+  });
+
+  /**
+   * * Add JSON field to component and set attribute.
+   */
+  mutateContent('adapt-contrib-accordion - update accordion._supportedLayout attribute to full-width', async () => {
+    accordions.forEach(accordion => {
+      /**
+      * ? Would this potentially break the layout of any course being migrated?
+      */
+      accordion._supportedLayout = 'full-width';
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-accordion - check accordion._supportedLayout atrribute is now full-width', async () => {
```

Comment:
The check should be that `_supportedLayout` is one of `full-width`, `half-width` or `both`

Reply:
Appreciated.